src/components/ExtensionAccountCheck.tsx

```diff
@@ -29,19 +29,21 @@ const ExtensionAccountCheck = ({ component }: Props): JSX.Element => {
   const { extensionExists, accountExists } = useKeyringContext();
   // TODO #176: Move this to a more generic error-show component
   const { sourceReady, targetReady } = useLoadingApi();
+  const isDevelopment = process.env.REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS === 'true';
```

Comment:
Let's make a separate flag for this? Now it will stop displaying the extension prompt on our current deployment, since we do load the dev accounts.